src/javascript/app/pages/user/account/statement/statement.init.js

```diff
@@ -53,9 +53,9 @@ const StatementInit = (() => {
     };
 
     const getAccountStatistics = () => {
-        BinarySocket.wait('landing_company').then(() => {
+        BinarySocket.wait('authorize').then(() => {
```

Comment:
no need to wait for authorize as statement page only loads if authorize comes, we already wait for that in binary loader